.github/workflows/ci_pipe.yml

```diff
@@ -165,11 +165,12 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        arch: ["amd64", "arm64"]
+        arch: ["amd64"]
```

Comment:
I don't think we want to remove `arm64`

Reply:
I didn't -- it's still in the include matrix below

Reply:
Did a force push maintaining the original style